libraries/python2-runtime/install.sh

```diff
@@ -9,3 +9,5 @@ apt-get install -u -y \
     python-pip
```

Comment:
talking with adnaan yesterday realized we probably don't need pip in the runtime image at all? anyone else have thoughts on that?

Reply:
Sounds good to me

Reply:
Also, kind of random but it seems like some versions (or more recent versions?) of apt-get don't have the --show-upgraded (-u) flag. My machine doesn't have it but I can see ubuntu 16.04 does

Reply:
Actually realized we need it to install packages into the runtime image (e.g. so tensorflow is installed)